api/client/utils.go

```diff
@@ -260,7 +260,7 @@ func (cli *DockerCli) monitorTtySize(id string, isExec bool) error {
 	sigchan := make(chan os.Signal, 1)
 	gosignal.Notify(sigchan, signal.SIGWINCH)
 	go func() {
-		for _ = range sigchan {
+		for range sigchan {
```

Comment:
I'm pretty sure I can ran into this too during some other PR. The latest gofmt wants this new format and drone fails because of it.